src/compas/tolerance.py

```diff
@@ -108,8 +114,9 @@ class Tolerance(Data):
     """
 
     def __new__(cls, *args, **kwargs):
-        if not hasattr(cls, "_instance"):
-            cls._instance = super(Tolerance, cls).__new__(cls)
+        if not isinstance(cls._instance, cls):
```

Comment:
mmm... not sure I understand here, under what scenario is this condition triggered?

Reply:
is it intended for inherited classes?

Reply:
I mean the only possible types for `_instance` are `Tolerance` and `NoneType`, no?